compose/reference/scale.md

```diff
@@ -13,3 +13,9 @@ Sets the number of containers to run for a service.
 Numbers are specified as arguments in the form `service=num`. For example:
 
     docker-compose scale web=2 worker=3
+
+>**Tip:** Alternatively, in
+[Compose file version 3.x](/compose/compose-file/index.md), you can specify
+[`replicas`](/compose/compose-file/index.md#replicas)
+under [`deploy`](/compose/compose-file/index.md#deploy) as part of the
+service configuration for [Swarm mode](/engine/swarm/).
```

Comment:
i think this should be indicated as today one of developer over here got mixed up because of that